subprojects/samples/samples.gradle.kts

```diff
@@ -36,6 +36,9 @@ gradlebuildJava {
 val integTestTasks: DomainObjectCollection<IntegrationTest> by extra
 integTestTasks.configureEach {
     libsRepository.required = true
+    inputs.files(gradleInstallationForTest.gradleSamplesDir.asFileTree)
```

Comment:
All integration test tasks need to depend on the samples, since some of them still use samples even if they are not in the `samples` subproject.

Reply:
👍